### PR TITLE
Fix RAW_MQ header inconsistencies after reblocking

### PR DESCRIPTION
**Pull request by ldgauthier**
I had a copy-paste guffaw when I deprecated the RAW_MQ header that changed the type to be incorrect.  After reblocking, the new header is correct, but conflicting and GenomicsDB is understandably not pleased.  Now there is a single, correct RAW_MQ header after reblocking.

### CODE REVIEW

**Comment by ldgauthier** [comment]
I can confirm that this fixes the production WGS reblocking issue.

**Comment by ldgauthier** [comment]
Lots of files are changed because I changed the key name for RAW_MQ to include `_DEPRECATED` and because I turned off the command line output in some of the test VCFs to save myself some pain later.

**Comment by ldgauthier** [comment]
@tlangs once we cut a release with this PR we can update and merge your reblocking workflow in dsde-pipelines.

File: src/main/java/org/broadinstitute/hellbender/tools/walkers/variantutils/ReblockGVCF.java [hunk 1]
```diff
@@ -158,6 +158,7 @@ public void onTraversalStart() {
         final Set<VCFHeaderLine> headerLines = new HashSet<>(inputHeaders);
         // Remove GCVFBlocks, legacy headers, and annotations that aren't informative for single samples
         headerLines.removeIf(vcfHeaderLine -> vcfHeaderLine.getKey().startsWith(GVCFWriter.GVCF_BLOCK) ||
+                (vcfHeaderLine.getKey().equals("INFO")) && ((VCFInfoHeaderLine)vcfHeaderLine).getID().equals(GATKVCFConstants.RAW_RMS_MAPPING_QUALITY_DEPRECATED) ||  //remove old (maybe wrong type) and add new with deprecated note
```
Comment (lbergelson) [inline src/main/java/org/broadinstitute/hellbender/tools/walkers/variantutils/ReblockGVCF.java]: I hate these classes so much.

**Pull request merged**